nnpdf_data/nnpdf_data/new_commondata/STAR_2015_1JET_200GEV/metadata.yaml

```diff
@@ -0,0 +1,50 @@
+setname: "STAR_2015_1JET_200GEV"
+
+nnpdf_metadata:
+  experiment: "STAR"
+  nnpdf31_process: "JETS"
+
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1850855"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1850855"
+  version: 1
```

Comment:
Please add arXiv reference.

Reply:
done in e5ed5285c69e8b8485ea5914099df9c8aadb4445